app/Http/Controllers/AccountController.php

```diff
@@ -40,19 +38,13 @@ public function changePassword(Request $request)
         $user = User::find(Auth::user()->id);
 
         #update password through user model.
-        if (empty($request->inputPasswordOld1) === False) {
-            if ($request->inputPasswordOld1 != $request->inputPasswordOld2)
-                return redirect()->back()->with('danger', 'Old passwords do not match. Try again');
-            elseif (Hash::check($request->inputPasswordOld1, Auth::user()->password) === false)
-                return redirect()->back()->with('danger', 'Incorrect password! Try Again.');
-            elseif ($request->newPW != $request->newPW2)
+            if ($request->newPW != $request->newPW2)
                 return redirect()->back()->with('danger', 'New passwords do not match. Try again');
             else {
                 $updated = True;
                 $user->password = Hash::make($request->newPW);
                 $user->save();
             }
-        }
 
         if ($updated === True)
             return redirect()->back()->with('status', 'Password change successful.');
```

Comment:
Instead of just redirecting back, could we go back to the user account page? Since they just changes their password, they probably don't want to be on the change password page anymore.